go.mod

```diff
@@ -91,9 +90,12 @@ replace (
 	github.com/fsnotify/fsnotify => github.com/adriansr/fsnotify v0.0.0-20180417234312-c9bbe1f46f1d
 	github.com/tonistiigi/fifo => github.com/containerd/fifo v0.0.0-20190816180239-bda0ff6ed73c
 	golang.org/x/tools => golang.org/x/tools v0.1.2
-	k8s.io/api => k8s.io/api v0.19.4
-	k8s.io/apimachinery => k8s.io/apimachinery v0.19.4
-	k8s.io/client-go => k8s.io/client-go v0.19.4
+	k8s.io/api => k8s.io/api v0.21.2
+	k8s.io/apimachinery => k8s.io/apimachinery v0.21.2
+	k8s.io/client-go => k8s.io/client-go v0.21.2
+	k8s.io/kubectl => k8s.io/kubectl v0.21.2
 )
 
+replace github.com/elastic/go-elasticsearch/v7 => github.com/elastic/go-elasticsearch/v7 v7.5.1-0.20210701130713-3c08ee738285
```

Comment:
why is sthis still necessary when using the released `7.13` version?

Reply:
It's not, after I had to update some `replace` directives to get new dependencies happy, i thought "well i guess i can just go to 7.14 if I'm already doing `replace` stuff". Happy to just use 7.13.1 though.

Reply:
got it; this is fine.